stdlib/experimental/geo/geo.flux

```diff
@@ -0,0 +1,134 @@
+// Provides functions for geographic location filtering and grouping based on S2 cells.
+package geo
+
+//
+// None of the builtin functions are intended to be used by end users.
+//
+
+// Check whether lat/lon is in a lat/lon box or center/radius circle.
+builtin containsLatLon
+
+// Calculates grid (set of cell ID tokens) for given box and according to options.
+builtin getGrid
+
+// Finds parent cell ID token for given cell ID at specified level.
+builtin getParent
+
+// Returns level of specified cell ID token.
+builtin getLevel
+
+//
+// Flux
+//
+
+// Gets level of cell ID tag `_ci` from the first record from the first table from the stream.
+_detectCiLevel = (tables=<-) => {
```

Comment:
Why do we only care about the first record of the first table?

Reply:
The assumption is that all records in all tables have cell id tag value of the same level.

Reply:
level here refers to the precision of the geohash.  We are going to make an assumption about the geohash...and try to infer it's precision based on the first record.